### PR TITLE
update SSP emissions files with anthropogenic forcing contributions

### PR DESCRIPTION
**Pull request by jedwards4b**
Updates for SSP emissions files.

### CODE REVIEW

**Comment by jedwards4b** [comment]
I've entered this change in the testdb https://csegweb.cgd.ucar.edu/testdb/cgi-bin/viewReleaseTag.cgi?tag_id=506 as cam_cesm2_1_rel_45

**Comment by jedwards4b** [comment]
@fvitt - yes they came from Simone Tilmes.

File: bld/namelist_files/use_cases/ssp126_cam6.xml [hunk 1]
```diff
@@ -32,9 +32,9 @@
...
          'so4_a1   -> $INPUTDATA_ROOT/atm/cam/chem/emis/emissions_ssp126/emissions-cmip6-ScenarioMIP_IAMC-IMAGE-ssp126-1-1_so4_a1_bb_surface_mol_175001-210101_0.9x1.25_c20190225.nc',
-         'so4_a2   -> $INPUTDATA_ROOT/atm/cam/chem/emis/emissions_ssp126/emissions-cmip6-ScenarioMIP_IAMC-IMAGE-ssp126-1-1_so4_a2_anthro-res_surface_mol_175001-210101_0.9x1.25_c20190225.nc',
-         'SOAG     ->   $INPUTDATA_ROOT/atm/cam/chem/emis/emissions_ssp126/emissions-cmip6-ScenarioMIP_IAMC-IMAGE-ssp126-1-1_SOAGx1.5_anthro_surface_mol_175001-210101_0.9x1.25_c20190225.nc',
-         'SOAG     ->  $INPUTDATA_ROOT/atm/cam/chem/emis/emissions_ssp126/emissions-cmip6-ScenarioMIP_IAMC-IMAGE-ssp126-1-1_SOAGx1.5_bb_surface_mol_175001-210101_0.9x1.25_c20190225.nc',
+         'so4_a2   -> $INPUTDATA_ROOT/atm/cam/chem/emis/emissions_ssp126/emissions-cmip6-ScenarioMIP_IAMC-IMAGE-ssp126-1-1_so4_a2_anthro-res_surface_mol_175001-210101_0.9x1.25_c20200403.nc',
+         'SOAG     ->  $INPUTDATA_ROOT/atm/cam/chem/emis/emissions_ssp126/emissions-cmip6-ScenarioMIP_IAMC-IMAGE-ssp126-1-1_SOAGx1.5_anthro_surface_mol_175001-210101_0.9x1.25_c20200403.nc',
```
Comment (cacraigucar) [inline bld/namelist_files/use_cases/ssp126_cam6.xml:36]: Does this file exist?  It does not exist on cheyenne nor in the svn repo.

Comment (jedwards4b) [reply bld/namelist_files/use_cases/ssp126_cam6.xml:36]: @cacraigucar Thanks for catching this, it was a typo and has now been corrected.  All the new files have been added to svn

**Pull request merged**